.github/workflows/ci.yml

```diff
@@ -0,0 +1,435 @@
+name: Alfresco Community Repo CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - feature/**
+      - fix/**
```

Comment:
I don't know the branching model adopted in this repo, but unless those branches are never merged into master using a PR, you are going to have duplicated builds (triggered both by pull_request and push on the same branch) - exclude those branches from pull_request or avoid at all having the push event here